holoviews/streams.py

```diff
@@ -52,7 +52,9 @@ def streams_list_from_dict(streams):
         if 'panel' in sys.modules:
             from panel.depends import param_value_if_widget
             v = param_value_if_widget(v)
-        if isinstance(v, param.Parameter) and isinstance(v.owner, param.Parameterized):
+        if (isinstance(v, param.Parameter)
+            and (isinstance(v.owner, param.Parameterized)
+                 or issubclass(v.owner, param.Parameterized))):
```

Comment:
Probably can simplify this too:

```suggestion
        if isinstance(v, param.Parameter) and v.owner is not None:
```

Reply:
Sure!